mappers/bluetooth_mapper/configuration/config_map_types.go

```diff
@@ -29,54 +29,151 @@ const (
 	BluetoothDivide   string = "Divide"
 )
 
-// DeviceProfile is structure to store in configMap
+// Twin provides a logical representation of control properties (writable properties in the
+// device model). The properties can have a Desired state and a Reported state. The cloud configures
+// the `Desired`state of a device property and this configuration update is pushed to the edge node.
+// The mapper sends a command to the device to change this property value as per the desired state .
+// It receives the `Reported` state of the property once the previous operation is complete and sends
+// the reported state to the cloud. Offline device interaction in the edge is possible via twin
+// properties for control/command operations.
+type Twin struct {
```

Comment:
Can we reference or reuse these structures from the api?

Reply:
Let me check the influence by import api